database/database.go

```diff
@@ -0,0 +1,104 @@
+package database
+
+import (
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"time"
+
+	"github.com/josephroberts/edge-node-manager/config"
+
+	"log"
+)
+
+func Start() error {
+	err := exec.Command(os.Getenv("GOPATH")+"/bin/tiedot", "-mode=httpd", "-dir="+filepath.Join(config.Persistant, config.Db), "-port="+config.DbPort).Start()
+	if err != nil {
+		log.Println("Failed to start database")
+		return err
+	}
+
+	time.Sleep(1 * time.Second)
+
+	return initialise()
+}
+
+func initialise() error {
+	resp, err := http.Get("http://localhost:" + config.DbPort + "/create?col=Devices")
+	if err != nil {
+		log.Println("Failed to create Devices collection")
+		return err
+	}
+	resp.Body.Close()
+
+	resp, err = http.Get("http://localhost:" + config.DbPort + "/index?col=Devices&path=applicationUUID")
+	if err != nil {
+		log.Println("Failed to add applicationUUID to index")
+		return err
+	}
+	resp.Body.Close()
+
+	resp, err = http.Get("http://localhost:" + config.DbPort + "/index?col=Devices&path=localUUID")
+	if err != nil {
+		log.Println("Failed to add localUUID to index")
+		return err
+	}
+	resp.Body.Close()
+
+	return nil
+}
+
+func Stop() {
+	resp, _ := http.Get("http://localhost:" + config.DbPort + "/shutdown")
```

Comment:
I am still swallowing some errors, not sure of the best way to catch

`return fmt.Errorf("Collection %s already exists", name)`